Tests/LLBuildManifestTests/LLBuildManifestTests.swift

```diff
@@ -62,7 +62,7 @@ final class LLBuildManifestTests: XCTestCase {
         manifest.addNode(.virtual("Foo"), toTarget: "main")
 
         let fs = InMemoryFileSystem()
-        try ManifestWriter(fileSystem: fs).write(manifest, at: "/manifest.yaml")
+        try ManifestWriter.write(manifest, at: "/manifest.yaml", fs)
```

Comment:
while we are here, should we call this something different than "Manifest"? the current name is pretty confusing with the package manifest